inputsources/composite/stackoutput.ftl

```diff
@@ -0,0 +1,43 @@
+[#ftl]
+
+[#-- Get stack output --]
+[#function azure_input_composite_stackoutput_filter outputFilter]
+  [#return 
+    {
+      "Subscription" : (outputFilter.Account)!accountObject.AWSId,
```

Comment:
Need to change the filter to AzureId I think

Reply:
As discussed, I had left this as AWSId to match my CMDB - the intent being to leave it as close to standard as possible. I have now updated CMDB to reflect the provider name, and so now have updated [it here](https://github.com/codeontap/gen3-azure/pull/56/commits/cf984568e4cdad3921656ec7359b7e35b4e1f7a3).